### DISCUSSION TITLE
feature proposal: config option for additional live stream

### DISCUSSION THREAD

**Discussion by unosonic** [Nachtzuster/BirdNET-Pi#216] [Ideas]
it would be nice to have a config option for an additional live stream on a remote icecast server, eg. if someone wants to make the live microphone input of a birdnet installation available to others. In scripts/livestream.sh the stream output to localhost is realized via ffmpeg. ffmpeg support multiple output streams in one command, eg:
```
ffmpeg -f alsa -i hw:0,0 -ac 2 -ar 48000 \
  -acodec libmp3lame -b:a 320k -content_type audio/mpeg \
  -f mp3 icecast://source:PASSWORD1@localhost:8000/mount1.mp3 \
  -acodec libmp3lame -b:a 160k -content_type audio/mpeg \
  -f mp3 icecast://source:PASSWORD2@icecast.server2:8000/mount2.mp3

```
just an idea.

**Comment by lolorc** [comment]
yep it would be nice to be more modular, but it would require running one ffmpeg instance per rtsp stream.

at the moment I use mediamtx to stream cam + mic to elsewhere,
I'm not using pulseaudio, I've replaced arecord by ffmpeg. ffmpeg is reading from mic, streaming to the local mediamtx mic rtsp and writing wav to be analyzed.
To this is working-ish, I want to add an external rtsp mic source, I'll have to run another ffmpeg process reading from rtsp and writing files but I don't know if birdnet_analysis can handle that.
